src/AddTodoForm.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 
-function AddTodoForm () {
+function AddTodoForm (props) {
 
+    function handleAddTodo(event) {
+        event.preventDefault();
+        const todoTitle = event.target.title.value;
+        console.log(todoTitle);
+        event.target.reset();
+        props.onAddTodo(todoTitle);
+    }
+    
     return (
-        <form>
+        <form onSubmit={handleAddTodo}>
             <label htmlFor='todoTitle'>Title</label>
-            <input id="todoTitle"></input>
+            <input id="todoTitle" name="title"/>
```

Comment:
Don't remember if we talked about this already, but it's nice to add placeholder text and the required attribute to make the input a bit more robust. Not necessary for the assignment so far, just good practice and stuff to keep in mind. 

```suggestion
            <input id="todoTitle" name="title" placeholder="Enter title here..." required />
```